pangeo_forge_esgf/dynamic_kwargs.py

```diff
@@ -13,21 +13,35 @@
     # TODO: I might not want to allow this in the ocean and ice fields. Lets see
 )
 
+daily_divisors = sorted(
+    [1, 30, 360, 365, 3653]
+    # TODO: It  might be more efficient to default to the underlying file chunks?
+)
+
 allowed_divisors = {
     "Omon": monthly_divisors,
     "SImon": monthly_divisors,
     "Amon": monthly_divisors,
+    "day": daily_divisors,
```

Comment:
Would these dividers also work generally for other daily data (see here: https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_table_id.json)?

Reply:
I honestly have no idea, I didn't realise there were different `day` table_id and can't find any documentation on what they represent. They might be the same but it seems a bit redundant. Happy to include them or not depending on how much you want to protect against potential future issues.

Reply:
I will try to check this PR out in the next few days, and maybe run some tests with the 'other' daily frequencies.